packages/web/src/pages/track-page/components/mobile/TrackHeader.module.css

```diff
@@ -54,8 +51,8 @@
   font-size: var(--font-l);
   font-weight: var(--font-medium);
   line-height: 24px;
+  justify-content: center;
   text-align: center;
-  margin-bottom: 16px;
```

Comment:
The general approach for TrackHeader was to remove the margins and borders from all of the individual section. Instead, I'm using `gap` on the main container to space the sections evenly, and applying a shared "withDivider" class to do top borders/padding on each section.

Reply:
this feels so satisfying :)